app/renderer/components/Inspector/ElementLocator.js

```diff
@@ -29,38 +56,37 @@ class ElementLocator extends Component {
       locatorTestValue,
       setLocatorTestStrategy,
       locatorTestStrategy,
+      driver,
       t,
     } = this.props;
 
-    const locatorStrategies = [
-      ['id', 'Id'],
-      ['xpath', 'XPath'],
-      ['name', 'Name'],
-      ['class name', 'Class Name'],
-      ['accessibility id', 'Accessibility ID'],
-      ['-android uiautomator', 'UIAutomator Selector (Android UiAutomator2)'],
-      ['-android datamatcher', 'DataMatcher Selector (Android Espresso)'],
-      ['-android viewtag', 'Android View Tag (Android Espresso)'],
-      ['-ios predicate string', 'Predicate String (iOS)'],
-      ['-ios class chain', 'Class Chain (iOS)'],
-    ];
-
-    return <div>
-      <Row>
-        {t('locatorStrategy')}
-        <Select className={InspectorStyles['locator-strategy-selector']}
-          onChange={(value) => setLocatorTestStrategy(value)}
-          value={locatorTestStrategy}>
-          {locatorStrategies.map(([strategyValue, strategyName]) => (
-            <Option key={strategyValue} value={strategyValue}>{strategyName}</Option>
+    return <Row justify="center">
+      <Radio.Group buttonStyle="solid"
+        className={InspectorStyles.locatorStrategyGroup}
+        onChange={(e) => setLocatorTestStrategy(e.target.value)}
+        defaultValue={locatorTestStrategy}
+      >
+        <Row justify="center">
+          {locatorStrategies(driver).map(([strategyValue, strategyName]) => (
+            <Radio.Button
+              className={InspectorStyles.locatorStrategyBtn}
+              value={strategyValue}
+              key={strategyValue}
+            >
+              {strategyName}
+            </Radio.Button>
           ))}
-        </Select>
-      </Row>
-      <Row>
-        {t('selector')}
-        <Input.TextArea className={InspectorStyles['locator-strategy-selector']} onChange={(e) => setLocatorTestValue(e.target.value)} value={locatorTestValue} />
-      </Row>
-    </div>;
+        </Row>
+      </Radio.Group>
+      <Input.TextArea
+        className={InspectorStyles.locatorSelectorTextArea}
```

Comment:
i think this field should have a label like 'Selector' or similar

Reply:
I had moved the label to the placeholder text instead, but to maintain consistency with the strategy label, I've now moved it back.